src/main/java/seedu/workbook/logic/commands/AddCommand.java

```diff
@@ -39,7 +40,13 @@ public class AddCommand extends Command {
             + PREFIX_TAG + "owesMoney";
 
     /** Message string displaying successful execution of the add command */
-    public static final String MESSAGE_SUCCESS = "New internship application added: %1$s";
+    public static final String MESSAGE_SUCCESS = "New internship application added: %1$s, " +
+            "check out some tips for the stage!";
+
+    /** Message string displaying successful execution of the add command, but stage is not in our pre-defined list */
+    public static final String MESSAGE_SUCCESS_STAGE_NO_TIPS = "New internship application added: %1$s, " +
+            "however, do you mean " + Stage.stageWithTipsToString() + " for the stages? We curated " +
```

Comment:
feel like this could be abit spammy for users since we are copy pasting all the stages we support. Ideally we would only show a supported stage if its "similar" to the user input, but that would be more complex...